.github/workflows/gradle.yml

```diff
@@ -19,4 +19,6 @@ jobs:
         with:
           java-version: 1.8
       - name: Build with Gradle
-        run: ./Source/DafnyRuntime/DafnyRuntimeJava/gradlew build
+        run: ./Source/DafnyRuntime/DafnyRuntimeJava/gradlew -p ./Source/DafnyRuntime/DafnyRuntimeJava build
```

Comment:
I'm not 100% sure but I suspect building "test" will invoke "build" automatically, so you can probably remove this step.

Reply:
Checked and it does. Edited.